KMB/make_KMB_info.py

```diff
@@ -154,22 +197,21 @@ def generate_content_cats(self, item):
         :param item: the KMBItem to analyse
         :return: list of categories (without "Category:" prefix)
         """
-        cats = []
+        cats = item.content_cats
 
         # depicted
         if item.fmis:
-            cats.append('Archaeological monuments in %s' % item.landskap)
-            cats.append('Archaeological monuments in %s County' % item.lan)
+            cats.add('Archaeological monuments in %s' % item.landskap)
+            cats.add('Archaeological monuments in %s County' % item.lan)
```

Comment:
Thoughts on categorization of archeo things: might be possible to go one level deeper from County -> Municipality. `Category:Archaeological monuments in Sweden by municipality` exists and is complete-ish, with over 200 subcats. In this case, if a yet nonexistent cat is used, it should be created automatically during the upload (or at least listed somewhere).

Interestingly, there's no similar structure for landskap -> socken, i.e. almost no `Category:Archaeological monuments in $socken socken`. How many archeo things are we even dealing with here? Because if it's a lot, then we might want to think about not overpopulating `Category:Archaeological monuments in $landskap`.